plugins/org.jkiss.dbeaver.ui.editors.sql/plugin.xml

```diff
@@ -1272,4 +1269,14 @@
             <objectType name="org.jkiss.dbeaver.ui.controls.resultset.IResultSetController"/>
         </generator>
     </extension>
+    
+	<extension point="org.eclipse.ui.propertyPages">
+		<page class="org.jkiss.dbeaver.ui.editors.sql.scripts.ScriptEditorPropertyPage" 
+              id="org.jkiss.dbeaver.ui.editors.sql.scripts.scriptEditorPropertyPage" 
+              name="%scriptEditorPropertyPage.name" nameFilter="*.sql">
+			<enabledWhen>
+				<adapt type="org.eclipse.core.resources.IFile" />
+			</enabledWhen>
+		</page>
```

Comment:
This formatting is weird, maybe we can fold the `class` attribute too, so they will align up perfectly?

Also, you're using tabs for some reason